bin/node/runtime/src/lib.rs

```diff
@@ -804,6 +806,25 @@ impl_runtime_apis! {
 			SessionKeys::decode_into_raw_public_keys(&encoded)
 		}
 	}
+
+	impl crate::Benchmark<Block> for Runtime {
```

Comment:
Should I hide the entire benchmarking pipeline behind a feature flag?

This means by default we should:

* Not expose a runtime API
* Not have CLI
* Not have benchmarking code in our runtime wasm

Is it possible?

Reply:
i think so; we don't really want benchmarking code making it on to the chain.